app/models/user.rb

```diff
@@ -3,11 +3,18 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
 
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
 
   def update_posts_counter
     update(posts_counter: posts.count)
   end
+
+  def update_comments_counter
+    update(comments_counter: comments.count)
+  end
 end
```

Comment:
- [ ] I noticed that no tests have been added for your `User Model`. So kindly ensure that you `Add unit specs for all the methods and validations` in your `User Model` :100:

Reply:
I have made the changes you suggested. Thanks for the kind feedback.